addons/sounds/functions/fnc_addSound.sqf

```diff
@@ -3,7 +3,14 @@
 Author: Crowdedlight
 			   
 File: fnc_addSound.sqf
-Parameters: pos, _unit
+Parameters: unit: Object to add the sound to
+			delayBetween: Number - delay in seconds between repeated plays
+			range: Number - range in meters at which the sound can be heard
+			repeat: Boolean - whether the sound should repeat
+			aliveCondition: Boolean - whether the sound should be removed on unit death
+			sound: String - the sound's shortID
+			delayInitial: Number - delay in seconds before the initial play
+			volume: Number - volume at which the sound should be played
```

Comment:
Seems like one tab too many, so they don't line up with the first ``unit`` param.

Reply:
Lines up on mine! 😄 
![image](https://github.com/Crowdedlight/Crows-Electronic-Warfare/assets/1899439/dec6a3eb-68fa-42bd-84dd-7d4038ad4e24)

Will fix 👍